rudder_airflow_provider/version.py

```diff
@@ -1 +1 @@
-__version__ = '1.1.0'
+__version__ = "1.1.0"
```

Comment:
@RanjeetMishra Are these changes intended?  If you are using any code formatter, it will nice to add a make command for the same

Reply:
we can just use pyproject.toml for now